test/Test/Record.purs

```diff
@@ -44,19 +48,20 @@ instance eqOuter ∷ Eq Outer where
         Just i1, Just i2 → i1.n == i2.n && i1.m == i2.m
         _, _ → false
 
-outerCodec ∷ JA.JsonCodec OuterR
+outerCodec ∷ CA.JsonCodec OuterR
 outerCodec =
-  JA.object "Outer" $ JAR.record
-    { a: JA.int
-    , b: JA.string
-    , c: JA.maybe innerCodec
+  CA.object "Outer" $ CAR.record
+    { a: CA.int
+    , b: CA.string
+    , c: CA.maybe innerCodec
     }
 
-innerCodec ∷ JA.JsonCodec InnerR
+innerCodec ∷ CA.JsonCodec InnerR
 innerCodec =
-  JA.object "Inner" $ JAR.record
-    { n: JA.int
-    , m: JA.boolean
+  CA.object "Inner" $ CAR.record
+    { n: CA.int
+    , m: CA.boolean
+    , o: CAR.Optional CA.boolean
```

Comment:
What do you think of an

    optional = Optional

so that you can write

    , o: CAR.optional CA.boolean

(which matches the other combinators a bit better)

Reply:
Doesn't bother me personally (plus the constructor version inlines without `purs-backend-es` :wink:) but I added it anyway :smile: